scripts/tests/consistency.sh

```diff
@@ -20,25 +20,28 @@ case $1 in
             #        it is not a default behaviour because Kagami resolves `build` path relative
             #        to the output file location
             temp_file="configs/swarm/docker-compose.TMP.yml"
-            full_cmd="$cmd_base --outfile $temp_file"
+            full_cmd="$cmd_base --out-file $temp_file"
 
             eval "$full_cmd"
             diff "$temp_file" "$target" || {
-                echo "Please re-generate \`$target\` with \`$cmd_base --outfile $target\`"
+                echo "Please re-generate \`$target\` with \`$cmd_base --out-file $target\`"
                 exit 1
             }
         }
 
+        genesis_keypair="--key-pair '{\"public_key\": \"ed01208BA62848CF767D72E7F7F4B9D2D7BA07FEE33760F79ABE5597A51520E292A0CB\",\"private_key\": {\"algorithm\": \"ed25519\",\"payload\": \"8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb\"}}'"
+        genesis_signature="--signature 9030303030303030302d303030302d303030302d303030302d3030303030303030303030308c4b28660000000000811b2c0400808ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb0101819b61a8c82fc6565c5acc7f92ca6dc07ccd39fcc47dee3d6f85d59ec7b52fe651b20b985b401546b49a50057b4311beaccb822b8ef2cf4e25ce3fab3769c70e"
```

Comment:
`iroha_swarm` needs to know only the signature

Reply:
I agree that `iroha_swarm` does not need to know full key-pair, but it does need to know the public key of genesis, isn't it? Otherwise we cannot verify it.